### PR TITLE
Add type annotations and tests for transaction isolation level

### PR DESCRIPTION
**Pull request by arturdryomov**

### CODE REVIEW

File: trino/transaction.py [hunk 1]
```diff
@@ -33,15 +34,15 @@ class IsolationLevel(object):
     SERIALIZABLE = 4
 
     @classmethod
-    def levels(cls):
-        return {k for k in cls.__dict__.keys() if not k.startswith("_")}
+    def levels(cls) -> Iterable[str]:
+        return {k for k, v in cls.__dict__.items() if not k.startswith("_") and isinstance(v, int)}
```
Comment (arturdryomov) [inline trino/transaction.py:38]: Without this check `levels`, `values` and `check` are returned as well.

File: tests/test_transaction.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
+from trino.transaction import IsolationLevel
```
Comment (ebyhr) [inline tests/test_transaction.py:1]: Add Apache license header.

**Review by ebyhr — APPROVED** [review]
Looks good except for a minor comment.

File: tests/test_transaction.py (hunk 2 above)
Comment (arturdryomov) [reply tests/test_transaction.py:13]: Done, thanks!

**Comment by ebyhr** [comment]
Merged, thanks!

**Pull request merged**